test/WebSites/ApiExplorerWebSite/Startup.cs

```diff
@@ -23,9 +23,23 @@ public void ConfigureServices(IServiceCollection services)
                 options.Conventions.Add(new ApiExplorerVisibilityDisabledConvention(
                     typeof(ApiExplorerVisbilityDisabledByConventionController)));
 
+                JsonOutputFormatter jsonOutputFormatter = null;
+                for (var i = 0; i < options.OutputFormatters.Count; i++)
+                {
+                    var formatter = options.OutputFormatters[i];
+                    jsonOutputFormatter = formatter as JsonOutputFormatter;
+                    if (jsonOutputFormatter != null)
+                    {
+                        break;
+                    }
+                }
```

Comment:
Thoughts on adding a `FormatterCollection<TFormatter>.Find<T>() where T : TFormatter` method (or `Get()` or `FindType()` or ...)? I did this twice in this PR and that may indicate users would do the same.